<launch>
    <param name="waypoints_file" type="string" value="$(find stretch_seeing_eye)/config/aws_hospital_wapoints/waypoints.csv" />
    <node pkg="stretch_seeing_eye" type="navigate_waypoints.py"  name="navigate_waypoints" output="screen">
        <param name="publish_frequency" type="double" value="30.0" />
    </node>
</launch>